aggregator.go

```diff
@@ -190,7 +189,7 @@ func prepareDBMigrations(dbStorage storage.OCPRecommendationsStorage) int {
 // prepareDB function opens a connection to database and loads all available
 // rule content into it.
 func prepareDB() int {
-	// TODO: when migrations for DVO will be available, update the code below
+	// TODO: when aggregator supports both storages at once, update the code below
```

Comment:
is there a task to support both storages? I would reference it here just so we know there's a plan

Reply:
yeah, [this one](https://issues.redhat.com/browse/CCXDEV-12316) I created yesterday. good idea

Reply:
yup, look at Dan's tasks